pan_controller:
    controller:
        package: dynamixel_controllers
        module: joint_torque_controller
        type: JointTorqueController
    joint_name: MX_Motor
    joint_speed: 0
    motor:
        id: 1
        init: 0
        min: 0
        max: 4095
